### PR TITLE
Update udata to the latest tagged version

### PR DESCRIPTION
**Pull request by magopian**

### CODE REVIEW

File: requirements.pip [hunk 1]
```diff
@@ -1,6 +1,6 @@
 uwsgi==2.0.21
 gevent==24.2.1
-udata==8.0.0
+udata==9.1.3
 udata-ckan==3.0.3
 udata-front==4.0.0
```
[the remark this answers is not in the document]
Comment (magopian) [reply requirements.pip:5]: Probably! I haven't tried it yet, but it makes sense indeed.

File: requirements.pip [hunk 2]
```diff
@@ -1,8 +1,8 @@
 uwsgi==2.0.21
 gevent==24.2.1
-udata==8.0.0
+udata==9.1.3
 udata-ckan==3.0.3
-udata-front==4.0.0
+udata-front==5.1.2
 udata-piwik==4.1.0
 udata-recommendations==3.1.5
 sentry-sdk[flask]==1.9.0
```
Comment (maudetes) [inline requirements.pip:8]:
```suggestion
```

File: requirements.pip [hunk 3]
```diff
@@ -1,8 +1,8 @@
 uwsgi==2.0.21
 gevent==24.2.1
-udata==8.0.0
+udata==9.1.3
 udata-ckan==3.0.3
-udata-front==4.0.0
+udata-front==5.1.2
 udata-piwik==4.1.0
 udata-recommendations==3.1.5
```
Comment (maudetes) [inline requirements.pip:7]:
```suggestion
udata-recommendations==3.1.6
```

**Review by maudetes — APPROVED** [review]
sentry-sdk should be removed since now pinned in udata directly.
udata-recommendations needs to be upgraded as well following https://github.com/opendatateam/udata-recommendations/pull/266

**Comment by magopian** [comment]
With the latest changes I could do a `docker compose up` without any issue, and using `docker exec -it docker-udata_udata_1 udata init` I made sure the latest changes to the fixtures are working properly 👍

**Pull request merged**